pkg/minikube/cluster/cluster.go

```diff
@@ -105,14 +105,14 @@ func StartHost(api libmachine.API, config cfg.MachineConfig) (*host.Host, error)
 func StopHost(api libmachine.API) error {
 	host, err := api.Load(cfg.GetMachineName())
 	if err != nil {
-		return errors.Wrapf(err, "Error loading host: %s", cfg.GetMachineName())
+		return errors.Wrapf(err, "Load: %s", cfg.GetMachineName())
```

Comment:
I'm wondering if we lose context here? I'd have to see the code to understand that this `Load: ...` error was at Stopping time?

Reply:
In this case, there is no contextual difference between "Error loading host" and "Load".